src/components/atoms/ActionButton.test.js

```diff
@@ -20,3 +25,12 @@ it("renders ActionButton in its withCounter state", () => {
   expect(screen.getByRole("button")).toHaveTextContent(WithCounter.args.text);
   expect(screen.getByRole("button").classList).toContain("bg-gray-light");
 });
+
+it("renders StartAgainButton in its icon state", () => {
+  render(<StartAgain {...StartAgain.args} />);
+  const buttonText = screen.getByText(StartAgain.args.text);
+  expect(buttonText).toBeTruthy();
```

Comment:
Is there a reason for assign this to a `const` var here?
Should the following not be simpler and cleaner?
```suggestion:-1+0
  expect(screen.getByText(StartAgain.args.text)).toBeTruthy();
```
or to be consistent with the other tests 
```
expect(screen.getByRole("button")).toHaveTextContent(StartAgain.args.text);
```

Reply:
Ty much cleaner from your changes.

Reply:
Maybe readability? but I do agree with you, make it one line looks cleaner.